File: b3324rec/app/src/main/java/com/example/b3324rec/MainActivity.java
package com.example.b3324rec;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;
import android.widget.RelativeLayout;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {
    Adapter adapter;
    ArrayList<LittleItem> littleItems;
    RecyclerView recyclerView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        recyclerView = findViewById(R.id.recycleView);

        littleItems = new ArrayList<>();
        littleItems.add(new LittleItem("Little 1"));
        littleItems.add(new LittleItem("Little 2"));
        littleItems.add(new LittleItem("Little 3"));
        littleItems.add(new LittleItem("Little 4"));

        recyclerView.setLayoutManager(new LinearLayoutManager(this));
        adapter = new Adapter(littleItems);
        recyclerView.setAdapter(adapter);

    }
}

File: l2092Recyc/app/src/main/java/com/example/l2092recyc/MainActivity.kt
package com.example.l2092recyc

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.MenuItem
import android.widget.Toast
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.google.android.material.navigation.NavigationView

class MainActivity : AppCompatActivity(), NavigationView.OnNavigationItemSelectedListener {

    var nav_view: NavigationView? = null
    var rcView: RecyclerView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        nav_view = findViewById(R.id.nav_view)
        nav_view?.setNavigationItemSelectedListener(this)

        rcView = findViewById(R.id.rcView)

        var list = ArrayList<ListItem>()
        list.add(ListItem(R.drawable.som, "Som", "Som presnyi"))
        list.add(ListItem(R.drawable.caras, "Caras", "Caras presnyi"))
        list.add(ListItem(R.drawable.shuca, "Shuca", "Shuca presnyi"))
        list.add(ListItem(R.drawable.nalim, "nalim", "nalim presnyi"))
        rcView?.hasFixedSize()
        rcView?.layoutManager = LinearLayoutManager(this)

        rcView?.adapter = MyAdapter(list, this)

    }

    override fun onNavigationItemSelected(item: MenuItem): Boolean {

        when(item.itemId){
            R.id.id_fish -> {
                Toast.makeText(this, "id_fish", Toast.LENGTH_LONG).show()
            }
            R.id.id_na -> {
                Toast.makeText(this, "id_na", Toast.LENGTH_LONG).show()
            }
            R.id.id_sna -> {
                Toast.makeText(this, "id_sna", Toast.LENGTH_LONG).show()
            }
            R.id.id_history -> {
                Toast.makeText(this, "id_history", Toast.LENGTH_LONG).show()
            }
        }
        return true
    }

}